pulp_rpm/app/viewsets/package.py

```diff
@@ -31,6 +31,7 @@ class Meta:
             "arch": ["exact", "in", "ne", "contains", "startswith"],
             "pkgId": ["exact", "in"],
             "checksum_type": ["exact", "in", "ne"],
+            "location_href": ["exact", "contains", "endswith"],
```

Comment:
LGTM functionality-wise, but I'm currently looking into whether we should ditch (or deprecate anyway) the `location_href` field in favor of something like "filename".

The reason being that `location_href` should never have existed in the current form to begin with, it stores the full path (think syncing), which doesn't really make much sense.

Reply:
Ah, I knew that location_href stored the full path. I think that storing just the filename makes sense. 

I'm pushing a change now that adds a "filename" filter. It looks like [relative_path on the content artifact has just the filename (no path)](https://github.com/pulp/pulp_rpm/blob/d8cb08dd9b6a400de77b86c7c2f20647b1df30e8/pulp_rpm/app/tasks/synchronizing.py#L1312-L1319) so I think it should work and it won't depend on location_href.